Tools/Scripts/webkitpy/port/darwin_testcase.py

```diff
@@ -40,6 +40,13 @@ def assert_skipped_file_search_paths(self, port_name, expected_paths, use_webkit
         port = self.make_port(port_name=port_name, options=MockOptions(webkit_test_runner=use_webkit2))
         self.assertEqual(port._skipped_file_search_paths(), expected_paths)
 
+    def test_lanes(self):
+        port = self.make_port()
+        self.assertEqual(sorted(port.sharding_groups().keys()), ['media'])
+        self.assertEqual('media', port.group_for_shard('media/something'))
+        self.assertEqual('media', port.group_for_shard('webaudio/something'))
+        self.assertEqual(None, port.group_for_shard('fast/something'))
```

Comment:
can we also test:

* `fast/media/something`,
* the WPT case,
* `media-session/something`?